src/shared/components/iframeLoader/IFrameLoader.tsx

```diff
@@ -30,7 +35,7 @@ export default class IFrameLoader extends React.Component<
     //NOTE: we need zindex to be higher than that of global loader
     render() {
         return (
-            <div style={{ position: 'relative' }}>
+            <div style={{ position: 'relative', width: this.props.width }}>
```

Comment:
It's odd that we apply the height to the iframe but the width to the div.  Let's discuss how to sort this out better.